rpc/jsonrpc/types/methods.go

```diff
@@ -809,6 +809,46 @@ type PurchaseTicketCmd struct {
 	TicketFee     *float64
 }
 
+// CreateUnsignedTicketCmd is a type handling custom marshaling and
+// unmarshaling of purchaseticket JSON RPC commands.
+type CreateUnsignedTicketCmd struct {
+	FromAccount   string
+	SpendLimit    float64 // In Coins
+	MinConf       *int    `jsonrpcdefault:"1"`
+	TicketAddress *string
+	NumTickets    *int
+	PoolAddress   *string
+	PoolFees      *float64
+	Expiry        *int
+	Comment       *string
+	TicketFee     *float64
+}
+
+// CreateUnsignedTicketResult models the data returned from the createmultisig
+// command.
+type CreateUnsignedTicketResult struct {
+	UnsignedTickets []string `json:"unsignedtickets"`
+	SplitTx         string   `json:"splittx"`
+}
+
+// NewCreateUnsignedTicketCmd creates a new PurchaseTicketCmd.
+func NewCreateUnsignedTicketCmd(fromAccount string, spendLimit float64, minConf *int,
```

Comment:
I'd prefer to not keep adding these constructor functions going forward, as they are extremely ugly and have to change or be replaced with alternative calls for any new options that must be added.  Create the command from the struct with named fields instead.